config/client/i18n.js

```diff
@@ -44,6 +44,10 @@ function format(value, format, languageCode) {
     return moment(value).format(format);
   }
 
+  if (typeof value === 'number' && format === 'number') {
```

Comment:
Can you think of a better name for a number format than `number`?

Reply:
I'm used to "number" in this context elsewhere. :-) 

https://codex.wordpress.org/Function_Reference/number_format_i18n
https://www.npmjs.com/package/@wordpress/i18n#_n